src/thrift-client/TgtInterfaceImpl.cpp

```diff
@@ -828,19 +910,33 @@ void StordVmdk::ScheduleNow(folly::EventBase* basep, StorRpcAsyncClient* clientp
 
 	basep->runInEventBaseThread([this, clientp, basep,
 			pending = std::move(pending)] () {
+		std::unique_ptr<std::vector<::hyc_thrift::WriteRequest>> write;
```

Comment:
If possible, can it just be seen by write Requests.

Reply:
this is just a variable declaration, the unique_ptr is not created here. Unique_ptr is created only when first write request is seen